README.adoc

```diff
@@ -16,31 +16,10 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Development
 
-To run this operator locally you need to have at least one Minishift profile started:
+toolchain-e2e repo makefile can be used as a starting point for setting up host and member operators for development.
 
-```bash
-$ minishift start --profile member
-```
-
-Then you can run the operator locally with the help of `operator-sdk` (you need version v0.19.2):
-
-```bash
-$ make up-local
-```
-
-That Makefile target takes care of additional several steps which can be executed separately:
-
-* logging as system:admin user: `$ make login-as-admin`
-* creating local test namespace: `$ make create-namespace`
-* deploy CRDs: `$ make deploy-crd`
+Following Makefile target takes care of additional several steps which can be executed separately:
 * building the project: `$ make build`
```

Comment:
I guess this is probably not very useful to keep anymore? If we're pointing to the dev targets in toolchain-e2e it's probably good enough? So we can just remove these 2 lines as well.
```suggestion
```